README.md

```diff
@@ -20,3 +20,75 @@ with a few commands. Need an ingress to serve traffic? Use Juju to deploy and
 integrate a range of ingresses, such as
 [Traefik](https://charmhub.io/traefik-k8s), and expose your application to
 external traffic in seconds.
+
+## Contributing
+
+Is there something missing, for example, a framework that you would like to use?
+PaaS App Charmer welcomes contributions! This section covers how to add a new
+integration and a new framework.
+
+### Add an Integration
+
+There are a few recommended steps to add a new integration which we'll go
+through below.
+
+1. Please write a proposal on the
+  [charm](https://discourse.charmhub.io/c/charm/41) topic on discourse. This
+  should cover things like:
+  * The integration you are intending to add
+  * For each of the frameworks that PaaS App Charmer supports:
+    - The commonly used package(s) to make use of the integration
+    - The environment variables, configuration etc. that would be made available
+      to the app
+    - An example for how to use the integration within an app
+  * The proposed implementation in `paas-app-charmer`. Take a look at
+    [`charm.py`](paas_app_charmer/_gunicorn/charm.py) for `gunicorn` based
+    frameworks for integration examples.
+1. Update the [tutorials](docs/tutorials) with the appropriate `fetch-lib`
+  command
+1. Raise a pull request to this repository adding support for the integration.
+1. Add a commented entry for `requires` to all the relevant Charmcraft
+  [templates](https://github.com/canonical/charmcraft/tree/main/charmcraft/templates)
+  for the new integration
+
+### Add a Framework
+
+There are a few recommended steps to add a new framework which we'll go through
+below.
+
+1. Please write a proposal on the
+  [charm](https://discourse.charmhub.io/c/charm/41) topic on discourse. This
+  should cover things like:
+  * The programming language and framework you are thinking of
+  * Create an example `rockcraft.yaml` file and build a working OCI image. To
+    see an example for `flask`, install the `latest/edge` version of `rockcraft`
+    and run `rockcraft init --profile flask-framework` and run
+    `rockcraft expand-extensions` and inspect the output.
+  * Create an example `charmcraft.yaml` file and build a working charm. To see
+    an example for `flask`, install the `latest/edge` version of Charmcraft
+    and run `charmcraft init --profile flask-framework` and run
+    `charmcraft expand-extensions` and inspect the output.
+  * How the configuration options of the charm map to environment variables,
+    configurations and so on of the app
```

Comment:
What's "and so on" here? It's suggesting some implied knowledge that the reader might not have, so it might be better to be explicit.